.github/workflows/release.yml

```diff
@@ -11,36 +23,50 @@ env:
 jobs:
   check-bump:
     runs-on: ubuntu-latest
-    if: github.ref_type == 'tag'
-
+    if: github.ref_type == 'tag' 
+      || (
+        github.event_name == 'workflow_dispatch'
+        && inputs.checkBump
```

Comment:
Question, how is this event triggered?

Reply:
Workflow dispatch can be done through the github ui, in the actions tab, or their REST API.